src/lib.rs

```diff
@@ -23,6 +23,21 @@ macro_rules! ready {
     }};
 }
 
+/// Simple macro to shadow a variable with a pinned reference.
```

Comment:
```suggestion
/// Pins a variable on the stack.
```

The purpose of this macro is safe (stack) pinning; shadowing is an implementation detail to make it safe.